src/components/Hoverable/index.js

```diff
@@ -41,6 +44,7 @@ class Hoverable extends Component {
         document.removeEventListener('mousedown', this.resetHoverStateOnOutsideClick);
         document.removeEventListener('touchstart', this.disableHover);
         document.removeEventListener('touchmove', this.enableHover);
+        document.removeEventListener('wheel', this.setScrollingAndStartUnsetListener);
```

Comment:
To respond to your question... it does not seem correct to listen to a `'wheel'` event as there's no guarantee it dispatches a `'scroll'` event. See: https://developer.mozilla.org/en-US/docs/Web/API/Element/wheel_event

**conclusion:** Use `'scroll'` not `'wheel'`